client/cli/src/commands/run_cmd.rs

```diff
@@ -459,6 +464,18 @@ impl CliConfiguration for RunCmd {
 		Ok(self.rpc_max_payload)
 	}
 
+	fn rpc_max_request_size(&self) -> Result<Option<usize>> {
+		Ok(self.rpc_max_request_size)
+	}
+
+	fn rpc_max_response_size(&self) -> Result<Option<usize>> {
+		Ok(self.rpc_max_response_size)
+	}
+
+	fn rpc_max_subscriptions_per_connection(&self) -> Result<Option<usize>> {
+		Ok(self.rpc_max_subscriptions_per_connection)
+	}
+
 	fn ws_max_out_buffer_capacity(&self) -> Result<Option<usize>> {
```

Comment:
Add a deprecation `eprintln!` here too? And/or docs?

Reply:
https://github.com/paritytech/substrate/blob/8392760e00162cd01d95e4534a1996963b3d8e92/client/service/src/lib.rs#L482-#L526

Added it :point_up: here because it's annoying to print if one doesn't has applied that `CLI option`